source/source.go

```diff
@@ -69,16 +70,11 @@ const (
 
 // Provider-specific annotations
 const (
-	// The annotation used for determining if traffic will go through Cloudflare
-	CloudflareProxiedKey        = "external-dns.alpha.kubernetes.io/cloudflare-proxied"
-	CloudflareCustomHostnameKey = "external-dns.alpha.kubernetes.io/cloudflare-custom-hostname"
+	// CloudflareProxiedKey The annotation used for determining if traffic will go through Cloudflare
+	CloudflareProxiedKey        = cloudflare.CloudflareProxiedKey
```

Comment:
Why redefining if we can just use the annotations package?

Reply:
Fixed